internetarchive/utils.py

```diff
@@ -398,3 +398,8 @@ def merge_dictionaries(dict0, dict1, keys_to_drop=None):
     # Items from `dict1` take precedence over items from `dict0`.
     new_dict.update(dict1)
     return new_dict
+
+
+def is_json(filename):
+    # Checks whether filetype is .json, else fallsback to .csv file format
+    return bool(re.fullmatch('^.*\.json$', filename))
```

Comment:
@maxz In this instance, the `bool()` is appropriate because a one-line utility `is_xyz()` function should return `True` or `False`.  Without `bool()` the function could return two different data types which is a bit confusing for the caller.
```
>>> import re
>>> type(re.fullmatch('^.*\.json$', "filename"))
<class 'NoneType'>
>>> type(re.fullmatch('^.*\.json$', "filename.json"))
<class 're.Match'>
```